storage/mysql/tree_storage.go

```diff
@@ -183,91 +188,56 @@ func encodeNodeID(n storage.NodeID) ([]byte, error) {
 	return marshalledBytes, nil
 }
 
-func (m *mySQLLogStorage) getDeleteUnsequencedStmt(num int) (*sql.Stmt, error) {
+func (m *mySQLTreeStorage) getStmt(statement string, num int) (*sql.Stmt, error) {
```

Comment:
We can possibly pull out all the statement building as a single unit. We'll need to make it more generic to support other databases anyway. No need to do now.